pkg/amphora/client.go

```diff
@@ -55,12 +79,47 @@ type Client struct {
 	HTTPClient http.Client
 }
 
-const secretShareURI = "/intra-vcp/secret-shares"
+const secretShareURI = "/secret-shares"
```

Comment:
Of course I understand why this _public_ interface is used at this point, since only there the corresponding function is provided, but I am still not sure if this should really be the case.

From Amphora Service [documentation](https://github.com/carbynestack/amphora/tree/master/amphora-service)
> Amphora provides three interfaces:
> 
>     User facing: Used by clients for secret sharing, accessing, and managing secret shares and their tags (/input-masks, /masked-input, /secret-shares).
>     Intra VCP: Used by services running in the same Virtual Cloud Provider to access locally stored secret shares, e.g. for ingesting into a computation (/intra-vcp).
>     Inter VCP: Used for communication with other Virtual Cloud Providers in the same Virtual Cloud (/inter-vcp).
> 
>     ⚠️ When setting up the system, it is necessary to limit access to non-public endpoints (/intra-vcp and /inter-vcp) to local services or the services provided by remote Virtual Cloud Providers.
> 

While the _user facing_ and _inter-vcp_ interfaces are supposed to be exposed to external parties and should be protected by authentication (and authorization) in the long run, _intra-vcp_ is considered as vcp internal only, blocked from external access and will eventually not require authentication.

With this, ephemeral would be required to authenticate to the `/secret-shares` endpoint in the future or new mechanisms to be added. 

Could it be more convenient to add similar functionality to the _inter-vcp_ interface, allowing to retrieve a list of secrets matching the specified criteria, but possibly providing only a list of IDs rather than a whole set of metadata that will be discarded anyway?